### DISCUSSION TITLE
Blogpost 3

### DISCUSSION THREAD

**Discussion by LeonRother** [LeonRother/SE_Platzverwaltung#4] [General]
In dieser Woche haben wir UML- und Sequenzdiagramme für bestimmte Use Cases erarbeitet.
Außerdem haben wir für die Use Cases [Platz buchen](https://github.com/LeonRother/SE_Platzverwaltung/blob/main/docs/UCRS_Platz_buchen.pdf) & [Defekt auflösen](https://github.com/LeonRother/SE_Platzverwaltung/blob/main/docs/UCRS_Defekt_auflösen.pdf) jeweils ein UCRS erstellt.

**Comment by SchloesserJonas** [comment]
Guten Abend,

Im SRS unter Punkt 2 beschreibt ihr, dass der Nutzer eine Bestätigung über die Buchung erhält.
Wie plant ihr diese Besätigung umzusetzen? (Email, Popup)

Welche Art von Error-Handling habt ihr für diese Funktion, auch hinsichtlich der Rückmeldung an den Nutzer, angedacht?

GaLiGrü 😊
Paula, Safae, Samuel, Valentin, Jonas

> Green Sprout 🌱

**Comment by GermanJesus-lul** [comment]
Hey,

Eure UCRS sehen schonmal gut aus. Aufgefallen ist mir "Defekt auflösen" UCRS nur, dass nach meinem Verständnis vor/bei dem Auswählen des Defektes die Defekte erst noch vom Server abgefragt werden müssen.
Oder wird diese Aktion in einem anderen Use Case, bspw. "Liste der Defekte anzeigen" übernommen?

Mit freundlichen Grüßen,
Julius und das Rezeptbuch Team
